src/systems/sprites.js

```diff
@@ -156,29 +129,46 @@ function enableSweepingEffect(comp) {
   const isFrozen = hoverableVisuals.el.sceneEl.is("frozen");
   const hideDueToPinning = !isSpawner && isPinned && !isFrozen;
   return hoverableVisuals.data.enableSweepingEffect && !hideDueToPinning;
-}
+};
 
-function createGeometry(maxSprites) {
+const createGeometry = maxSprites => {
+  // TODO: Consider to use InstancedBufferGeometry which would be more efficient and performant
```

Comment:
Yeah I think the sprite system predates that being added to Three (or possibly we just didn't know about it at the time). It definitely seems like the right way to handle this. We can probably tackle that whenever we actually get to reworking object menus, since that is by far the biggest usage of sprites.